engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -524,10 +525,10 @@ object CromwellApiServiceSpec {
         sender ! response
       case AbortWorkflow(id, manager) =>
         val message = id match {
-          case ExistingWorkflowId =>
-            WorkflowStoreEngineActor.WorkflowAborted(id)
+          case ExistingWorkflowId => WorkflowStoreEngineActor.WorkflowAborted(id)
           case AbortedWorkflowId =>
             WorkflowAbortFailed(id, new IllegalStateException(s"Workflow ID '$id' is in terminal state 'Aborted' and cannot be aborted."))
+          case WorkflowId(_) => throw new Exception("Something untoward happened")
```

Comment:
This is "clearly the test author didn't think this could possibly happen, so this should probably blow up: :)